app/src/main/java/org/gdg/frisbee/android/api/model/Gde.java

```diff
@@ -14,11 +17,12 @@ public Gde createFromParcel(Parcel in) {
             return new Gde[size];
         }
     };
-    private String product, name, address, email, socialUrl;
+    private String name, address, email, socialUrl;
+    private List<String> product = new ArrayList<>();
     private double lat, lng;
 
     private Gde(Parcel in) {
-        product = in.readString();
+        in.readStringList(product);
```

Comment:
Does this expect product ArrayList not to be null? 
You can use this instead http://developer.android.com/reference/android/os/Parcel.html#createStringArrayList()